src/ansys/dpf/post/modal_mechanical_simulation.py

```diff
@@ -6,7 +6,7 @@
 """
 from typing import List, Union
 
-from ansys.dpf import core
```

Comment:
you can have cyclic in static and harmonic analysis too

Reply:
@cbellot000 Indeed, I was waiting on validation of the API before propagating it to every method of `StaticMechanicalSimulation`, `ModalMechanicalSimulation` and `HarmonicMechanicalSimulation`. Will do!